_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -369,6 +369,8 @@ public function get_module( $request ) {
 				&& ( new Status() )->is_offline_mode()
 			) {
 				$module['activated'] = false;
+			} elseif ( Jetpack::is_module_active( $request['slug'] ) ) { // TODO: Move into `Jetpack::get_module()`?
```

Comment:
Do we need to check that `slug` is set before we try to use it?

Reply:
I don't think we do, since we're already relying on it to be present a few lines above:

https://github.com/Automattic/jetpack/blob/1527e0be4d25312d5fb4672a3f856e11ad6d84db/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php#L360-L364